Mon May 11 14:57:00 CEST 2020, Following exception was thrown: Specified itemID does not exist: 5
se.kth.iv1350.pos.integration.InvalidItemIDException: Specified itemID does not exist: 5
	at se.kth.iv1350.pos.controller.Controller.registerItem(Controller.java:57)
	at se.kth.iv1350.pos.view.View.registerItem(View.java:56)
	at se.kth.iv1350.pos.view.View.sampleExecution(View.java:43)
	at se.kth.iv1350.pos.startup.Main.main(Main.java:27)
